x-pack/plugins/observability_ai_assistant/public/functions/visualize_esql.tsx

```diff
@@ -127,19 +127,14 @@ export function VisualizeESQL({
         },
       };
 
-      const chartSuggestions = lensHelpersAsync.value.suggestions(context, dataViewAsync.value);
+      const chartSuggestions = lensHelpersAsync.value.suggestions(
+        context,
+        dataViewAsync.value,
+        [],
+        preferredChartType
+      );
       if (chartSuggestions?.length) {
-        let [suggestion] = chartSuggestions;
```

Comment:
ℹ️ I just removed the logic from here and moved it to the api which I think is the best to host it